docs/content/preview/drivers-orms/java/postgres-jdbc.md

```diff
@@ -89,34 +87,37 @@ If you are using [Gradle](https://docs.gradle.org/current/samples/sample_buildin
 implementation 'org.postgresql:postgresql:42.2.14'
 ```
 
-### Step 2: Connect to your cluster
+Install the added dependency using `mvn install`.
+
+### Step 2: Set up the database connection
 
 After setting up the dependencies, implement a Java client application that uses the PostgreSQL JDBC driver to connect to your YugabyteDB cluster and run a query on the sample data.
 
 Java applications can connect to and query the YugabyteDB database using the `java.sql.DriverManager` class. The `java.sql.*` package includes all the JDBC interfaces required for working with YugabyteDB.
 
 Use the `DriverManager.getConnection` method to create a connection object for the YugabyteDB Database. This can be used to perform DDLs and DMLs against the database.
 
-Example PostgreSQL JDBC URL for connecting to YugabyteDB can be seen below.
+
+| JDBC parameter | Description | Default |
+| :------------- | :---------- | :------ |
+| hostname | Hostname of the YugabyteDB instance | localhost |
+| port | Listen port for YSQL | 5433 |
+| database | Database name | yugabyte |
+| user | User connecting to the database | yugabyte |
+| password | User password | yugabyte |
+
+Following is the PostgreSQL JDBC URL format for connecting to YugabyteDB:
 
 ```java
```

Comment:
not java - doesn't work without the quotes and interprets the slashes as a comment
also, below the example URLs are just jdbc:postgresql, ie without the first slashes whats up with that

Reply:
I don't think we need those slashes. Let me get rid of them at other places too . 
I was able to run it without those slashes- `String yburl = "jdbc:yugabytedb://127.0.0.1:5433/yugabyte?user=yugabyte&password=yugabyte&load-balance=true";`